subprojects/docs/src/docs/release/notes.md

```diff
@@ -192,6 +192,7 @@ The following are the newly deprecated items in this Gradle release. If you have
 - Moved classes `RhinoWorkerHandleFactory` and `RhinoWorkerUtils` into internal package.
 - Removed `RhinoWorker`.
 - Removed constructor `EarPluginConvention(Instantiator)`.
+- Removed the Gradle UI module. All classes for this feature have been removed as well as all leftovers supporting class from the Open API partly removed due to deprecation in Gradle 2.0.
```

Comment:
I believe users rather know the UI module as the [Gradle GUI](https://docs.gradle.org/current/userguide/tutorial_gradle_gui.html).